frontend/packages/ceph-storage-plugin/src/utils/bucket-class.ts

```diff
@@ -0,0 +1,34 @@
+import { bucketClassNameRegex, timeUnits } from '../constants/bucket-class';
+
+const hasConsecutivePeriodsOrHyphens = (name: string): boolean => {
+  for (let i = 0; i < name.length - 1; i++) {
+    if (name.charAt(i) === '.') {
+      if (name.charAt(i + 1) === '.') return true;
+    }
+    if (name.charAt(i) === '-') {
+      if (name.charAt(i + 1) === '-') return true;
+    }
+  }
+  return false;
+};
```

Comment:
you can use a regex - `/(-|\.)\1/g` , just for suggestion.

Reply:
changed in previous PR